### PR TITLE
CentOS: Clarify kernel update

### PR DESCRIPTION
**Pull request by MaEtUgR**
I want to clarify some points that come to my mind while I'm setting up new devices

### CODE REVIEW

File: en/getting_started/CentOS.md [hunk 1]
```diff
@@ -65,12 +65,15 @@ SDL2 is used for joystick support.
 sudo yum install SDL2 SDL2-devel -y
 ```
 
-### Update Kernel to support all USB Joysticks
+### Update Kernel (optional)
 
-Chances are that your USB Joystick does not get recognized under CentOS.
-In our tests the following joysticks did not work out of the box:
+The two resons why we update the kernel are:
```
Comment (hamishwillee) [inline en/getting_started/CentOS.md]: resons >reasons

File: en/getting_started/CentOS.md [hunk 2]
```diff
@@ -65,12 +65,15 @@ SDL2 is used for joystick support.
...
+- Better touch screen responsiveness
+- Correct recognition of some USB devices especially joysticks
```
Comment (hamishwillee) [inline en/getting_started/CentOS.md]: add comma after devices.

**Comment by hamishwillee** [comment]
Thanks @MaEtUgR ! A couple of typos, but I'll fix as a post process.

**Comment by hamishwillee** [comment]
FYI, done in https://github.com/mavlink/qgc-dev-guide/commit/43cf88409b1e59516c8a8280fd6f6c0b03d64926

**Comment by MaEtUgR** [comment]
Thanks a lot @hamishwillee ! You're the best lector!

**Pull request merged**